nautobot/docs/user-guide/platform-functionality/jobs/job-scheduling-and-approvals.md

```diff
@@ -59,14 +58,52 @@ The queue of jobs that need approval can be found under `Approvals > Approval Da
 
 ### Approval via the API
 
-Approvals can also be given via the REST API. The endpoints to approve, deny, and dry run a scheduled job are found on the scheduled job endpoint under `approve`, `deny`, and `dry-run`, respectively.
+Approvals can also be given via the REST API. The endpoints to approve, deny, comment and pending-approvals are found on the approval workflow stage endpoint under `approve`, `deny`, `comment` and `pending-workflows` respectively. You may also include a comment in the request data when approving or denying a workflow.
+
+#### Approve/Deny a Workflow
+
+```no-highlight
+curl -X POST \
+-H "Authorization: Token $TOKEN" \
+-H "Content-Type: application/json" \
+-H "Accept: application/json; version=1.3; indent=4" \
+-d '{"comment": "Approved for deployment"}' \
+http://nautobot/api/extras/approval-workflow-stages/$APPROVAL_WORKFLOW_STAGE_ID/approve
+```
+
+```no-highlight
+curl -X POST \
+-H "Authorization: Token $TOKEN" \
+-H "Content-Type: application/json" \
+-H "Accept: application/json; version=1.3; indent=4" \
+-d '{"comment": "Deny reason"}' \
+http://nautobot/api/extras/approval-workflow-stages/$APPROVAL_WORKFLOW_STAGE_ID/deny
+```
+
+#### Comment on an Approval Workflow Stage
+
+The `comment` endpoint allows a user to attach a non-approval comment to a specific stage within an approval workflow. This endpoint does not change the state of the stage, and is intended for adding informational messages, questions, or updates related to the approval process.
+
+- This will attach a comment to the specified stage.
+- The stage state will remain unchanged.
+- The user must have the `change_approvalworkflowstage` permission.
 
 ```no-highlight
 curl -X POST \
 -H "Authorization: Token $TOKEN" \
 -H "Content-Type: application/json" \
 -H "Accept: application/json; version=1.3; indent=4" \
-http://nautobot/api/extras/scheduled-jobs/$JOB_ID/approve?force=true
+-d '{"comments": "Waiting for additional testing."}' \
+http://nautobot/api/extras/approval-workflow-stages/$APPROVAL_WORKFLOW_STAGE_ID/comment
 ```
 
-The approval endpoint additionally provides a `force` query parameter that needs to be set if a job is past its scheduled datetime.
+#### List Pending Approvals
+
+Retrieves a list of approval workflow stages that are pending approval for the current user. This can be done through the `pending-approvals` endpoint, which returns all workflow stages that are awaiting approval from the authenticated user.
```

Comment:
Can we make this a filter/filterset parameter on the regular list endpoint rather than adding a separate endpoint for this?